src/Application/Renderer/Template/Action/SourcecodeHandler.php

```diff
@@ -37,7 +37,7 @@ public function __construct(Analyzer $analyzer)
      */
     public function __invoke(Action $action)
     {
-        $artifact = $action->getRenderPass()->getDestination() . '/' . ltrim($action->getDestination(), '\\/');
+        $artifact = $action->getRenderContext()->getDestination() . '/' . ltrim($action->getDestination(), '\\/');
```

Comment:
is it artifact or artefact? both are being used, and if they have the same meaning I'd suggest to unify naming.